backend/api/src/main/kotlin/software/shonk/adapters/incoming/ShorkInterpreterController.kt

```diff
@@ -216,6 +216,26 @@ fun Route.configureShorkInterpreterControllerV1() {
         val errors = shorkUseCase.getCompilationErrors(compileErrorsRequest.code)
         call.respond(mapOf("errors" to errors))
     }
+
+    get("/lobby/{lobbyId}/settings") {
+        val lobbyId =
+            call.parameters["lobbyId"]?.toLongOrNull()
+                ?: return@get call.respond(HttpStatusCode.BadRequest)
```

Comment:
This responds with Bad Request which does not conform to the spec. According to the spec the response for a non existent lobby should be Not Found.